P/Powsybl/build_tarballs.jl

```diff
@@ -70,7 +73,7 @@ install_license $WORKSPACE/srcdir/powsybl.jl/LICENSE.md
 include("../../L/libjulia/common.jl")
 platforms = vcat(libjulia_platforms.(julia_versions)...)
 
-filter!(p -> arch(p) == "x86_64" && os(p) ∈ ("windows", "linux", "macos"), platforms)
+filter!(p -> arch(p) == "x86_64" && os(p) ∈ ("windows", "linux", "macos") || arch(p) == "aarch64" && occursin(os(p), "macos"), platforms)
```

Comment:
```suggestion
filter!(p -> arch(p) == "x86_64" && os(p) ∈ ("windows", "linux", "macos") || arch(p) == "aarch64" && Sys.isapple(p), platforms)
```
I think you're missing some parentheses to correctly group the conditions?

Reply:
Thanks for isapple. For parenthesis I think it is okay logically ? A && B || C && D should be evaluated as ( x86_64 && 3os ) || (aarch64 && macos)